README.md

```diff
@@ -68,4 +68,96 @@ By default all image uploads are uploaded to a cloud provider of your choice. I
 
 ## Installation
 
-Coming soon...
+### Server
+
+* Clone this repo!
+* Download and install https://getcomposer.org/download/
+
+```
+php composer.phar install
+```
+
+* Follow instructions @ https://laravel.com/docs/5.7/homestead to setup homestead
```

Comment:
- I'd include a link to [Laravel Valet](https://laravel.com/docs/5.7/valet), as it's an even easier way to get Laravel apps up and running than Homestead (imo). Good to have both though.